sdks/wasm/Makefile

```diff
@@ -110,20 +110,22 @@ main.exe: $(APP_SOURCES)
 mini_tests.dll: $(MINI_TEST_SOURCES) mini-test-runner.cs
 	$(CSC) $(CSC_FLAGS) /unsafe -target:library -out:$@ -define:__MOBILE__,ARCH_32 $(BCL_DEPS) /r:$(WASM_BCL_DIR)/nunitlite.dll  $(MINI_TEST_SOURCES) mini-test-runner.cs
 
-binding_tests.dll: WebAssembly.Bindings.dll $(BINDING_TEST_SOURCES)
-	$(CSC) $(CSC_FLAGS) /unsafe -target:library -out:$@ /r:WebAssembly.Bindings.dll $(BCL_DEPS) /r:$(WASM_BCL_DIR)/nunitlite.dll $(BINDING_TEST_SOURCES)
+binding_tests.dll: .stamp-bindings $(BINDING_TEST_SOURCES)
+	$(CSC) $(CSC_FLAGS) /unsafe -target:library -out:$@ /r:$(WASM_BCL_DIR)/WebAssembly.Bindings.dll $(BCL_DEPS) /r:$(WASM_BCL_DIR)/nunitlite.dll $(BINDING_TEST_SOURCES)
 
-WebAssembly.Bindings.dll: bindings.cs
-	$(CSC) $(CSC_FLAGS) /target:library -out:$@ /r:$(WASM_BCL_DIR)/mscorlib.dll bindings.cs
+.stamp-bindings: bindings.cs
+	$(CSC) $(CSC_FLAGS) /target:library -out:$(WASM_BCL_DIR)/WebAssembly.Bindings.dll /r:$(WASM_BCL_DIR)/mscorlib.dll bindings.cs
```

Comment:
Can't this simply be:
$(WASM_BCL_DIR)/WebAssembly.Bindings.dll: bindings.cs

Reply:
fixed